kogito-bom/pom.xml

```diff
@@ -170,6 +170,23 @@
         <version>${version.org.kie.kogito}</version>
         <classifier>sources</classifier>
       </dependency>
+      
+      <!-- Data Index -->
+      <dependency>
+        <groupId>org.kie.kogito</groupId>
+        <artifactId>data-index-service</artifactId>
+        <version>${project.version}</version>
```

Comment:
I believe the version should be ${version.org.kie.kogito} and same goes for all newly added dependencies

Reply:
fixed